lib/beaker/shared/subcommands_util.rb

```diff
@@ -1,22 +1,33 @@
+require 'rake'
+
 module Beaker
   module Shared
     # Methods used in execution of Subcommands
     # - should we write the config?
-    # - reset ARGV 
+    # - reset ARGV
     # - execute Beaker
     module SubcommandsUtil
       CONFIG_PATH = ".beaker/config"
-    
+      RAKE_FILE = "Rakefile"
```

Comment:
As I understand it, rake looks for one of the following rakefiles in the current directory:

rakefile, Rakefile, rakefile.rb, Rakefile.rb

I feel like we should be taking this into account when searching/grepping the rakefile, and return whichever one we find for appending our require to